### ISSUE TITLE
Firefox NVDA doesn't read content by arrow down after closing the print modal

### ISSUE THREAD

**Issue by antonpalinau** [nvaccess/nvda#10450]
### Steps to reproduce:
1. Open Firefox
2. Browse to https://codepen.io/antonpalinau/pen/MWWOoBJ
3. click on print button
4. close print modal using tab and enter
5. press arrow down to read content

### Actual behavior:
Content is not announced.

### Expected behavior:
Hello world and button any focusable should be announced by NVDA.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2019.1

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Firefox 70.0

**Comment by DrSooom** [comment]
Workaround: Press the ALT key twice after the print dialog was closed.

**Comment by lukaszgo1** [comment]
cc @jcsteh

**Comment by JulienCochuyt** [comment]
Tested with installed NVDA 2019.2.1, Windows 10 version 1903 (18362.418) and Firefox 70.0.1 (64 bits)
Reproducible on any web page by invoking the Print dialog with `control+p`.
Seems we are missing a gain focus event upon closing the dialog, or failing to handle it properly.

**Comment by Cleversn** [comment]
Hi, just to report this also happens when closing other dialogs, such as open and save as. You may test by pressing control+o, then esc, then an arrow key.

**Comment by DrSooom** [comment]
Furthermore this issue exists for many years. And after "a little bit" of searching, I found it: NVDA issue #2497 and [Firefox bug 782547](https://bugzilla.mozilla.org/show_bug.cgi?id=782547).

**Comment by JulienCochuyt** [comment]
Thanks @DrSooom: I just commented on the FF bug tracker to let them know we're still affected.

cc @jcsteh

**Comment by ehollig** [comment]
As noted by @jcsteh in https://github.com/nvaccess/nvda/issues/2497#issuecomment-228218468

> There's nothing more we can do here; if/when Mozilla fix this, it will behave as expected.

CC @MarcoZehe 
Closing

**Comment by JulienCochuyt** [comment]
@ehollig shouldn't we leave this issue open to later check, once the issue gets fixed in Firefox, that  NVDA integrates correctly with it?

**Comment by ehollig** [comment]
Marking as needsExternalFix

**Comment by jcsteh** [comment]
> @ehollig shouldn't we leave this issue open to later check, once the issue gets fixed in Firefox, that NVDA integrates correctly with it?

I tend to do this for some issues where the fix might be unclear or obscure. In this case, though, the fix will either work or it won't, and I'll be testing it against NVDA anyway, so I don't see any need to track this here.

**Comment by jcsteh** [comment]
This is (finally!) fixed for Firefox 74 (currently available for testing in Firefox Nightly).
